systemvm/debian/opt/cloud/bin/setup/bootstrap.sh

```diff
@@ -85,12 +85,18 @@ config_guest() {
           sed -i "s/%/ /g" $CMDLINE
           ;;
      kvm)
-          # Configure hot-plug
-          modprobe acpiphp || true
-          modprobe pci_hotplug || true
+          # Configure kvm hotplug support
+          if grep -E 'CONFIG_HOTPLUG_PCI=y|CONFIG_HOTPLUG_PCI_ACPI=y' /boot/config-`uname -r`; then
+            log_it "acpiphp and pci_hotplug module already compiled in"
+          else
+            modprobe acpiphp 2> /dev/null && log_it "acpiphp module loaded" || true
+            modprobe pci_hotplug 2> /dev/null && log_it "pci_hotplug module loaded" || true
+          fi
+
           sed -i -e "/^s0:2345:respawn.*/d" /etc/inittab
           sed -i -e "/6:23:respawn/a\s0:2345:respawn:/sbin/getty -L 115200 ttyS0 vt102" /etc/inittab
-          systemctl enable --now qemu-guest-agent
```

Comment:
Minor nit: The `--now` does the same, enables and starts it.

Reply:
@rhtyd yes normally it should but qemu-guest-agent it a sysvinit and unfortunately with this redirect there is no startup. 

```sh
[root@kvm08 /]# /lib/systemd/systemd-sysv-install --help
systemd-sysv-install version 1.7.4 - Copyright (C) 1997-2000 Red Hat, Inc.
This may be freely redistributed under the terms of the GNU Public License.

usage:   systemd-sysv-install <enable|disable|is-enabled> [name]
```